chop-wood.lic

```diff
@@ -31,7 +31,7 @@ class ChopWood
       snapshot = Room.current.id
       wait_for_script_to_complete('safe-room')
       walk_to(snapshot)
-      @equipment_manager.wield_weapon(@lumber_implement)
+     # @equipment_manager.wield_weapon(@lumber_implement)
```

Comment:
:x: Just remove it - no need to comment out.

Reply:
This also doesn't seem like the right solution. I dont see the connection between the log you provided and removing this line, can you explain?

Reply:
How about we stow the cutting implement before `wait_for_script_to_complete('safe-room')` then this would continue as expected?